src/medCore/gui/toolboxes/medRegistrationSelectorToolBox.cpp

```diff
@@ -207,14 +178,13 @@ void medRegistrationSelectorToolBox::changeCurrentToolBox(int index)
     }
 }
 
-
 //! Clears the toolbox.
 void medRegistrationSelectorToolBox::clear(void)
```

Comment:
It actually clears *inputData*, which is not used in registration.
So I think adding this line is useless.
Btw, I had a look also at *medToolBox::clear()*, which really does a clearance and not a reset as other *clear()* functions do. If we have time/courage we should change that.